aait/backend/group-2B/Application/Features/Posts/Requests/Queries/GetPostsByUserIdRequest.cs

```diff
@@ -0,0 +1,12 @@
+using MediatR;
+using SocialSync.Application.DTOs.PostDtos;
+
+namespace SocialSync.Application.Features.Posts.Requests.Queries;
+
+public class GetPostsByUserIdRequest: IRequest<List<GeneralPostDto>>{
+    public int UserId;
```

Comment:
It would be better if we make userId a property,
making userId a property aligns with common coding conventions and design patterns. It provides a consistent approach for accessing and manipulating the identifier across different classes and codebases.

Possible change:

`public class GetPostsByUserIdRequest: IRequest<List<GeneralPostDto>>{
    public int UserId { get; set; }
}
`

Reply:
Fixed